overlay/hooks/entrypoint-pre.d/10_generate_config.sh

```diff
@@ -0,0 +1,195 @@
+#/bin/bash
+
+set -e
+
+ZONEPATH="/etc/bind/cache/"
+ZONETEMPLATE="/etc/bind/cache/zone.tmpl"
+CACHECONF="/etc/bind/cache.conf"
+USE_GENERIC_CACHE="${USE_GENERIC_CACHE:-false}"
+LANCACHE_DNSDOMAIN="${LANCACHE_DNSDOMAIN:-cache.lancache.net}"
+CACHE_ZONE="${ZONEPATH}$LANCACHE_DNSDOMAIN.db"
+RPZ_ZONE="${ZONEPATH}rpz.db"
+DOMAINS_PATH="/opt/cache-domains"
+
+reverseip () {       
+    local IFS        
+    IFS=.            
+    set -- $1        
+    echo $4.$3.$2.$1 
+}                    
+
+export GIT_SSH_COMMAND="ssh -o UserKnownHostsFile=/dev/null -o StrictHostCACHE_IDENTIFIERChecking=no"
```

Comment:
I think `CACHE_IDENTIFIER` here should be `Key`.
```suggestion
export GIT_SSH_COMMAND="ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no"
```